driver/src/main/java/org/neo4j/driver/internal/messaging/BoltProtocol.java

```diff
@@ -168,6 +169,10 @@ else if ( BoltProtocolV42.VERSION.equals( version ) )
         {
             return BoltProtocolV42.INSTANCE;
         }
+        else if ( BoltProtocolV43.VERSION.equals( version ) )
```

Comment:
is there more cleanup we can do for removing 4.0? possibly not since its the first in the series

Reply:
I kept the 4.0 there since we don't have a final decision about the versions we will support in 4.3